docs/manual.tex

```diff
@@ -182,7 +191,7 @@ \subsection{Multiplication}
 \end{lstlisting}
 
 Specialized to operate on units it's only slightly more involved. The `Typeable` contexts arise because they are
-required to ensure that an appropriate `KnownVariant` instance will exist. Again, `Typeable` isntances exist for all
+required to ensure that an appropriate `KnownVariant` instance will exist. Again, `Typeable` instances exist for all
```

Comment:
Is there some way to run a spellchecker over this? It won't catch me not knowing how to conjugate "has", but it might still be helpful. ;)